_posts/2018-10-12-strimzi-0.8.0-release.md

```diff
@@ -0,0 +1,164 @@
+
+<!--more-->
+
+# Accessing Kafka cluster from outside without TLS
+
+The previous release added the possibility to access the Kafka cluster from the outside.
+The Kafka clients were able to exchange messages when not running on the same Kubernetes/OpenShift cluster where Kafka was deployed but from the outside.
+The prerequisite was the TLS protocol enabled on client side because all the provided `listeners` were accessible only thorugh encryption.
+With this new release, all the `listeners` have TLS enabled by default but it is possible to disable it by setting the `tls` property to `false`.
+Of course, disabling TLS is possible only for `loadbalancer` and `nodeport` listeners but not on OpenShift `route` for which the TLS is mandatory.
+
+Here's an example snippet that exposes the Kafka cluster using `loadbalancer` and TLS disabled:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    ...
+    listeners:
+      external:
+        type: loadbalancer
+        tls: false
+    ...
+```
+
+# Handling certificates renewal and expirations
+
+The cluster CA and clients CA certificates are only valid for a limited time period, known as the validity period which is defined as the number of days since the certificate was generated.
+For auto-generated certificates, it is possible to configure the validity period by setting the `validityDays` property.
+Manually-installed CA certificates should have their own validity period defined.
+
+When a CA certificate expires, the certificates that it has signed will fail validation, even if they were previously valid.
+This means that, when replacing a CA certificate, you must also replace all other certificates signed by it.
+When the replacement of a CA certificate is in progress, it is necessary for peers to trust certificates signed by either the old or the new CA.
+This ensures the continued operation of the cluster.
+
+To allow the renewal of CA certificates without a loss of service, the Cluster Operator will initiate certificate renewal before the old CA certificates expire.
+It is possible to configure the renewal period by setting the `renewalDays` property.
+
+Here's an example snippet that defines auto-generated certificates validity and renewal period:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    ...
+    clusterCA:
+      generateCertificateAuthority: true
+      validityDays: 730
+      renewalDays: 60
+    clientsCA:
+      generateCertificateAuthority: true
+      validityDays: 365
+      renewalDays: 15
+    ...
+```
+
+The renewal period is measured backwards, from the expiry date of the current certificate.
+
+![Certificates renewal]({{ "/assets/2018-10-12-certs-renewal.png" }})
+
+# A new operator in town: Mirror Maker!
+
+The Strimzi operators family has now a new baby: the Kafka Mirror Maker operator.
```

Comment:
:)  "...family has a new baby"

I would capitalise "Operator" to match the docs.